response_operations_ui/controllers/survey_controllers.py

```diff
@@ -100,3 +100,39 @@ def update_survey_details(survey_ref, short_name, long_name):
         raise ApiError(response)
 
     logger.debug('Successfully updated survey details', survey_ref=survey_ref)
+
+
+def get_legal_basis_list():
+    logger.debug('Retrieving legal basis list')
+    url = f'{app.config["SURVEY_SERVICE_URL"]}/legal-bases'
+    response = requests.get(url, auth=(app.config['SURVEY_SERVICE_USERNAME'], app.config['SURVEY_SERVICE_PASSWORD']))
```

Comment:
Will this not always be the same as the basic auth username and password?

Reply:
No, I asked Dave about this and he said each service would have it's own username password